exercises/ruby-1-a/README.md

```diff
@@ -4,17 +4,18 @@ This is Part 1 of our research into how people write Ruby differently. This exer
 
 ## Instructions
 
-Your task is to determine which word in a sentence has the most vowels (a, e, i, o, u). A sentence is defined as a [Ruby string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
-
-Your job is to define a method with the signature `word_with_most_vowels`, which takes a sentence, and returns the word with the most vowels.
+Your task is to define a ShoppingCart that can receive a hash of items and has two methods: `total_amount` and `items_list`. The method `total_amount` will return the sum of all the items’ prices. The method `list` will return a string with the name of the items separated by commas.
 
 For example:
 
 ```ruby
-word_with_most_vowels("Hi there aunt jeanie")
-// Returns: "jeanie"
-```
+shopping_cart = ShoppingCart.new({ potatoes: 10.00, bananas: 30.00 })
```

Comment:
This covers that comment made with a code comment, but we should repeat it in code, making the initial declaration in words as we do "items".